### PR TITLE
Reword E2E training and inference tips in the vision guides

### PR DESCRIPTION
**Pull request by sayakpaul**
Reference: https://github.com/huggingface/datasets/pull/5188#discussion_r1012148730

### CODE REVIEW

**Review by nateraw — APPROVED** [review]
Thanks!

**Review by lhoestq — APPROVED** [review]
thanks :)

File: docs/source/object_detection.mdx [hunk 1]
```diff
@@ -154,9 +154,14 @@ You can verify the transform works by visualizing the 10th example:
...
-[this notebook](https://nbviewer.org/github/NielsRogge/Transformers-Tutorials/blob/master/YOLOS/Fine_tuning_YOLOS_for_object_detection_on_custom_dataset_%28balloon%29.ipynb).
-
 <div class="flex justify-center">
     <img src="https://huggingface.co/datasets/nateraw/documentation-images/resolve/main/visualize_detection_example_transformed_2.png">
 </div>
+
+<Tip>
+
+Now that you know how to process a dataset for object detection, learn
+[how to train an object detection model](https://nbviewer.org/github/NielsRogge/Transformers-Tutorials/blob/master/YOLOS/Fine_tuning_YOLOS_for_object_detection_on_custom_dataset_%28balloon%29.ipynb)
```
Comment (stevhliu) [inline docs/source/object_detection.mdx:164]:
```suggestion
[how to train an object detection model](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/YOLOS/Fine_tuning_YOLOS_for_object_detection_on_custom_dataset_(balloon).ipynb)
```

**Review by stevhliu — APPROVED** [review]
Thanks for the updates!

**Pull request merged**